src/utils/components/DiskModal/utils/submit.ts

```diff
@@ -0,0 +1,127 @@
+import produce from 'immer';
+
+import { V1beta1DataVolume } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { UploadDataProps } from '@kubevirt-utils/hooks/useCDIUpload/useCDIUpload';
+import { getName } from '@kubevirt-utils/resources/shared';
+import {
+  getBootDisk,
+  getDataVolumeTemplates,
+  getDisks,
+  getVolumes,
+} from '@kubevirt-utils/resources/vm';
+import { generatePrettyName, isEmpty } from '@kubevirt-utils/utils/utils';
+import { isRunning } from '@virtualmachines/utils';
+
+import { DEFAULT_DISK_SIZE } from './constants';
+import { getEmptyVMDataVolumeResource, hotplugPromise, produceVMDisks } from './helpers';
+import { V1DiskFormState } from './types';
+
+const makeItBootableVolume = (vm: V1VirtualMachine, diskName: string) => {
```

Comment:
add return type
and please change this function name to something clear,
`applyDiskAsBootable` or `applyBootableDisk` (these are just suggestion based on what I understand from this function, feel free to create another that might fit better)